src/box/lua/schema.lua

```diff
@@ -34,10 +34,12 @@ ffi.cdef[[
     typedef struct tuple box_tuple_t;
     typedef struct iterator box_iterator_t;
 
-    /** \cond public */
     box_iterator_t *
-    box_index_iterator(uint32_t space_id, uint32_t index_id, int type,
-                       const char *key, const char *key_end);
+    box_index_iterator_after(uint32_t space_id, uint32_t index_id, int type,
+                             const char *key, const char *key_end,
+                             const char *packed_pos,
+                             const char *packed_pos_end);
+    /** \cond public */
```

Comment:
Nit: These comments (`cond public`) make no sense in `ffi.cdef`. Let's drop them all from this block while we are at it.

Reply:
Done